jest.config.js

```diff
@@ -1,3 +1,23 @@
+function resolveTsconfigPathsToModuleNameMapper() {
+	const tsconfigPath = './tsconfig.json';
+	const { paths } = require(tsconfigPath).compilerOptions;
+
+	const moduleNameMapper = {
+		'^.+\\.module\\.(css|sass|scss)$': 'identity-obj-proxy',
+	};
+
+	Object.entries(paths).forEach(([alias, path]) => {
+		// "@appServices/*" becomes "@appServices/(.*)"
+		const key = alias.replace('/*', '/(.*)');
+		// "assets/src/application/services/*" becomes "<rootDir>/assets/src/application/services/$1"
+		const value = '<rootDir>/' + path[0].replace('/*', '/$1');
+
+		moduleNameMapper[key] = value;
+	});
+
+	return moduleNameMapper;
+}
+
```

Comment:
Aliases for tests are now calculated from TS config

Reply:
oh cool 👍 
good stuff

Reply:
yes, nice one :tada: